pandas/_libs/missing.pyx

```diff
@@ -290,12 +291,22 @@ def _create_binary_propagating_op(name, divmod=False):
 
     def method(self, other):
         if (other is C_NA or isinstance(other, str)
-                or isinstance(other, (numbers.Number, np.bool_))):
+                or isinstance(other, (numbers.Number, np.bool_, np.int64, np.int_))
```

Comment:
Why are the `np.int64, np.int_` needed? `numbers.Number` should already deal with those I think, and also adding only int64 seems strange